applications/DAOsign.md

```diff
@@ -0,0 +1,183 @@
+to name a few.
+Also, Eugene is a founder at consideritdone.tech and daosign.org
+
+**Ramil Amerzyanov:** Solution Architect, CTO
+Background: 
+12+ years in building complex enterprize solutions (all SDLC aspects), including 7+ years in Blockchain industry.
+Past projects include:
+- contribution to Ethereum  standards development
+- tech leadership for https://docsend.com/view/fjrvjtyzgm7wgrkj projects
+
+**Oleksandra Burmenska:** Director of Business Development and Partnerships
+Background:
+Engineering, MBA, and Psychology diplomas. Over a decade of experience in IT: development and managerial positions, last five years focused on the Blockchain industry. 
+Experience with enterprises and startup companies, scaling teams, and building blockchain products.
+Previous blockchain projects include:
+- successfully managed ICOs during 2017-2019 (example: Õpet – Medium ) 
+- contribution to Polywrap, NEAR, Axelar, GnosisSafe, Avalanche ecosystems
+- contribution to nodle.com ecosystem: managing the block explorer development (before it moved to Subscan), leading the Landslide - Avalanche subnet development,etc. 
+
+**Misha Kushka**: Tech Lead and Blockchain Developer
+Background: 6+ years of professional experience as a developer, 4+ years of professional experience in the Blockchain field, 4+ years experience as a lead
+
+### Team Code Repos
+
+- https://github.com/ConsiderItDone
+- https://github.com/DAOsign
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- [Eugene Fine](https://github.com/ConsiderItDone)
+- [Ramil Amerzyanov](https://github.com/ramilexe)
+- [Oleksandra Burmenska](https://github.com/Burmenska)
+- [Misha Kushka](https://github.com/kushkamisha)
+
+### Team LinkedIn Profiles (if available)
+
+- [Eugene Fine](https://www.linkedin.com/in/eugenefine/)
+- [Ramil Amerzyanov](https://www.linkedin.com/in/ramil-amerzyanov/)
+- [Oleksandra Burmenska](https://www.linkedin.com/in/oleksandra-burmenska/)
+- [Misha Kushka](https://www.linkedin.com/in/mkushka/) 
+
+
+## Development Status :open_book:
+
+We are currently working on a tech demo (kind of a pre-beta version) and non-blockchain version of DAOsign is available here: https://testnet.daosign.org
+
+## Development Roadmap :nut_and_bolt:
+
+**Overview**
+
+Total Estimated Duration: 2 months
+Full-Time Equivalent (FTE): 2 FTE per month
+Total Costs: 48,100 USD
+
+Milestone 1 - Smartcontracts. 
+Total Costs: 40,300 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can upload/generate the agreement, add signers/observers and complete the signature process with storing signature proofs either in test or mainnet, which will show how the new functionality works. |
```

Comment:
What do you mean by mainnet here? Polkadot has no smart contract functionality.

Reply:
Updated. Also, I squashed smart contract deliveries into 1 line with description